QEfficient/utils/generate_inputs.py

```diff
@@ -72,9 +81,15 @@ def prepare_pytorch_inputs(self):
         )
 
         if self.full_batch_size:
-            inputs["input_ids"] = input_ids
-            inputs["position_ids"] = torch.arange(input_len).view(1, input_len)
+            # Feed input without padding (CB pt forward pass fails if padding exists in position_ids)
             inputs["batch_index"] = torch.arange(1).view(-1, 1)
+            if self.num_logits_to_keep is not None:

```

Comment:
Could you explain why this change is necessary? 
Was inputs["input_ids"] = input_ids omitted by mistake?